File: test/fixtures/python/put_basic_auth_json_data.py
import requests

headers = {
    'Content-Type': 'application/x-www-form-urlencoded',
}

data = '{"admins":{"names":[], "roles":[]}, "readers":{"names":["joe"],"roles":[]}}'

response = requests.put('http://localhost:28139/test/_security', headers=headers, data=data, auth=('admin', '123'))

File: test/fixtures/r/put_basic_auth_json_data.r
library(httr)

headers = c(
  `Content-Type` = "application/x-www-form-urlencoded"
)

data = '{"admins":{"names":[], "roles":[]}, "readers":{"names":["joe"],"roles":[]}}'

res <- httr::PUT(url = "http://localhost:28139/test/_security", httr::add_headers(.headers=headers), body = data, httr::authenticate("admin", "123"))
